src/components/TodoList/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,58 @@
+import classNames from 'classnames';
+import { Todo } from '../../../types/Todo';
+
+type Props = {
+  todo: Todo;
+  removeTodo: (TodoId: number) => Promise<void>;
+  selectedId: number[];
+};
+
+export const TodoItem: React.FC<Props> = ({
+  todo,
+  removeTodo,
+  selectedId,
+}) => {
+  return (
+    <div
+      data-cy="Todo"
+      className={classNames(
+        'todo',
+        {
+          completed: todo.completed,
+        },
+      )}
+      key={todo.id}
+    >
+      <label className="todo__status-label">
+        <input
+          data-cy="TodoStatus"
+          type="checkbox"
+          className="todo__status"
+          defaultChecked
+        />
+      </label>
+
+      <span data-cy="TodoTitle" className="todo__title">{todo.title}</span>
+      <button
+        type="button"
+        className="todo__remove"
+        data-cy="TodoDeleteButton"
+        onClick={() => {
+          removeTodo(todo.id);
+        }}
+      >
+        ×
+      </button>
+      <div
+        data-cy="TodoLoader"
+        className={classNames(
+          'modal overlay',
+          { 'is-active': selectedId.includes(todo.id) },
```

Comment:
Better create a separate variable for `selectedId.includes(todo.id)` and use it here.